### ISSUE TITLE
Auto Updater?

### ISSUE THREAD

**Issue by Vastrix** [jaruba/PowderPlayer#138]
Hey,  
  
Does PowderPlayer auto update?  
If not -> Feature request :)  
  
If So, maybe show it somewhere through an option for disabling it?  
Also, maybe show the current PowderPlayer version somewhere? :)  
  
Cheers o/

**Comment by jaruba** [comment]
Nope, no auto-updater yet, pre v1.00 it had an auto-updater but there were a lot of issues with it. I should definitely look into newer solutions for it though.

There is an "About" in the Context Menu of the Player that shows the current version, probably not placed very well as this was requested a lot since it's there.

**Comment by Vastrix** [comment]
ah aight :)
I suggest changing it to the Settings pane?  

Thanks!

**Comment by jaruba** [comment]
Let's leave this open till i actually make an auto updater again

**Comment by jaruba** [comment]
Related ticket for Powder Web: https://github.com/jaruba/PowderWeb/issues/8
